bin/update-browsers.sh

```diff
@@ -3,12 +3,12 @@
 # Update the browser regex in the version check script.
 #
 # Uses a browserslist query to generate this data.
+# Needs you to run `npx browserslist-useragent-regexp "defaults, unreleased versions"` before first use
```

Comment:
Hmm, why is this required, because npx prompts for install?

Reply:
so I have npm/npx installed. But the command needed a one time only "y" manual enter to work